src/data/reducers/assets.js

```diff
@@ -23,6 +23,13 @@ const list = (state = [], action) => {
 
 const status = (state = {}, action) => {
   switch (action.type) {
+    case assetActions.CLEAR_ASSETS_STATUS:
+      return {};
```

Comment:
I added this because the status message is what is being used to determine the styling of the StatusAlert -- there is a short amount of time where the alert will show up before the delete request has completed and returned a response, this could cause the styling to show incorrectly briefly (green first to red failure or vice versa).  By clearing the status after the close of the alert, the next time the alert is triggered it will show the warning/"Deleting asset" text before showing the correct status (yellow to red/green).